newIDE/app/src/MainFrame/index.js

```diff
@@ -1895,6 +1899,8 @@ class MainFrame extends React.Component<Props, State> {
                   (this._resourceSourceDialogs[resourceSource.name] = dialog)
                 }
                 i18n={i18n}
+                loadPreferencesValues={loadPreferencesValues}
```

Comment:
Better wrap this in a `PreferencesContext.Consumer` to avoid having polluting MainFrame props.

Reply:
Yes, that's better. I will update it.

Reply:
I have updated the PR with your suggestions, please review it :)